cedric-ms/src/main/java/org/dice_research/sask/cedric_ms/CedricMsController.java

```diff
@@ -0,0 +1,50 @@
+package org.dice_research.sask.cedric_ms;
+
+import java.io.IOException;
+import javax.servlet.http.HttpServletResponse;
+import org.apache.log4j.Logger;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * The class provides the REST interface for the microservice.
+ * 
+ * @author Kevin Haack
+ *
+ */
+@RestController
+public class CedricMsController {
+
+	/**
+	 * The logger.
+	 */
+	private Logger logger = Logger.getLogger(CedricMsController.class.getName());
+
+	/**
+	 * Represent a simple version of extraction, without configuration.
+	 * 
+	 * @param input
+	 *            The input to extract.
+	 * @return The extraction result.
+	 */
+	@RequestMapping("/extractSimple")
+	public String extractSimple(String input) {
+		CedricMsService service = CedricMsService.getInstance();
+		return service.extract(input);
+	}
+
+	@ExceptionHandler
+	void handleIllegalArgumentException(IllegalArgumentException e, HttpServletResponse response) throws IOException {
+		this.logger.error("FRED-microservice IllegalArgumentException: " + e.getMessage());
```

Comment:
logger.error need to be Cedric-microservice

Reply:
Should somebody change something here? If so who? Please suggest actions for each review

Reply:
@KHaack here "this.logger.error("FRED-microservice IllegalArgumentException ..." FRED-microservice should be changed Cedric-microservice ... I mentioned you because you coded this extractor :)

Reply:
[Testing.docx](https://github.com/dice-group/sask/files/1937607/Testing.docx)
I have tested the functionalities and added the screenshots in the file.